docs/site/home.html

```diff
@@ -96,26 +96,35 @@ <h3 class="block-title">What is it?</h3>
         </div>
 
         <div hljs language="js" ng-show="selected == 'compute engine'">
-// Auth is handled automatically inside GCE.
-var gcloud = require('gcloud');
-var dataset = gcloud.datastore.dataset();
-var productKey = dataset.key(['Product', 123]);
+// Authentication is automatic
+// inside Google Compute Engine.
+var gcloud = require('gcloud')({
+  projectId: 'grape-spaceship-123'
+});
+
+var gcs = gcloud.storage();
 
-dataset.get(productKey, function(err, entity) {
-  console.log(err, entity);
+var backups = gcs.bucket('backups');
+backups.upload('db.zip', function(err, file) {
```

Comment:
I don't think it's important to live up to a fictional scenario, so much that it becomes believable/copy-and-paste-able. I think a good example has to be somewhat real to make the user feel comfortable with the concept, but also needs to show what can be done with our API to spark some ideas for how the user can use it in their own project.